core-java-modules/core-java-string-operations-3/src/main/java/com/baeldung/tostring/StringCastUtils.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.tostring;
+
+public class StringCastUtils {
+    public static String castToString(Object object) {
+        if (object instanceof String)
```

Comment:
Let's make sure we include parenthesis for the if statements. Generally we follow that rule for our samples.